tests/impact/client/entities/test_project.py

```diff
@@ -0,0 +1,72 @@
+from tests.impact.client.fixtures import *
+from tests.impact.client.helpers import (
+    create_project_content_entity,
+    create_project_entity,
+)
+from modelon.impact.client import ContentType
+from tests.files.paths import SINGLE_FILE_LIBRARY_PATH
+
+
+class TestProject:
+    def test_get_project_contents(self, project):
+        contents = project.entity.get_contents()
+        assert contents == [
+            create_project_content_entity(
```

Comment:
Instead of having asserts like these where the expected content class is created by some logic somewhere else it would probably be clearer to make asserts on the fields, this will make it easier to read the code and see what is being tested. Like:
```
assert len(contents) == 1
assert contens[0].id == "..."
```